manager/src/main/resources/db/migration/0.1/V0.1.16__add_shard_router_hostname.sql

```diff
@@ -0,0 +1,2 @@
+alter table SHARD
+    add column "router_canonical_hostname" varchar(255);
```

Comment:
How do you expect to migrate the existing environments? 
Is this covered by a future PR?

@robpblake 
I think we might decide to expose a `registerShard` endpoint to do the wiring and provide similar information. Wdyt?

Reply:
as we have only one shard for dev and one shard for stable, I'll login into the cluster and update the database tables accordingly. 

We still have to design/implement all the components to terraform a shard - I'd expect this column will be populated during that process when a new shard will be registered using the admin api/whatever

Reply:
@r00ta I think we should automate the setting the value of the `router_canonical_hostname` in all environments. In particular in `app-interface`, we don't have the ability to log in to the DB and insert data by hand.

The column should also have the `not null` constraint as a Shard without a `router_canonical_hostname` entry is not valid.

I'd suggest we follow the approach that we've taken previously so something like:

```
quarkus.flyway.placeholders.shard-router-canonical-hostname=${SHARD_ROUTER_CANONICAL_HOSTNAME}
```
in `application.properties` and then have the migration reference the placeholder.
We can then update each environment configuration (including `app-interface`) to update the data and then set the column to have the `not null` constraint afterwards.

cheers,
Rob